src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -1065,6 +1065,12 @@ - (void)openCommissioningWindowWithDiscriminator:(NSNumber *)discriminator
     [baseDevice openCommissioningWindowWithDiscriminator:discriminator duration:duration queue:queue completion:completion];
 }
 
+- (void)verifyAttestationWithQueue:(dispatch_queue_t)queue completion:(MTRDeviceAttestationVerificationHandler)completion
+{
+    auto * baseDevice = [self newBaseDevice];
+    [baseDevice verifyAttestationWithQueue:queue completion:completion];
```

Comment:
If going with the helper class approach it could be directly created here be given the MTRBaseDevice, rather than having the code itself live inside MTRBaseDevice.